build/build-changelog.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$UPDATE = " - Added support for latest API updates.";
+$path = __DIR__ . '/../CHANGELOG.md';
+
+$diff = shell_exec("git status --porcelain");
```

Comment:
Okay, that's good. When I was running the command it was erroring out with `porcelain` not defined. Must've misspelled it!